### DISCUSSION TITLE
anyone else having speed breaks extended on their own

### DISCUSSION THREAD

**Discussion by icywings11** [flybywiresim/aircraft#4981] [General]
First- AMAZING WORK!!!!!!!
Quick question- is it a known bug that the speed breaks extend on their own at times? I am a real world Boeing pilot. I have left the computer during cruise and come back to find the spoilers were extended. I've also noticed they extend during taxi on their own. I see them extended, but the speed break lever is in the retract position. When I move the lever full to retract they retract. 
Thanks again FBW!!

**Comment by tracernz** [comment]
Check if you have anything bound to spoilers on your keyboard, joystick etc. even if (especially if!) they're not actually used.

**Reply by icywings11** [reply]
Yes I think this is my issue. Thanks
